go/vt/worker/topo_utils.go

```diff
@@ -38,14 +38,14 @@ var (
 	// have been taken out by previous *Clone or *Diff runs.
 	// Therefore, the default for this variable must be higher
 	// than vttablet's -health_check_interval.
-	waitForHealthyTabletsTimeout = flag.Duration("wait_for_healthy_rdonly_tablets_timeout", 60*time.Second, "maximum time to wait at the start if less than --min_healthy_rdonly_tablets are available")
+	waitForHealthyTabletsTimeout = flag.Duration("wait_for_healthy_tablets_timeout", 60*time.Second, "maximum time to wait at the start if less than --min_healthy_tablets are available")
```

Comment:
Although I agree this is the right change to make, this will break existing use cases for anyone who is setting the old flag name. @michael-berlin / @sougou is that going to cause problems?